WIP uac (#4296)
libuv/libuv: docs/src/process.rst

             * search for the exact file name before trying variants with
             * extensions like '.exe' or '.cmd'.
             */
            UV_PROCESS_WINDOWS_FILE_PATH_EXACT_NAME = (1 << 7),
            /*
             * Run the subprocess as administrator. This option is only meaningful on
             * Windows systems. On Unix it is silently ignored.

>              * This flag is mutually exclusive with UV_PROCESS_DETACHED. UV_EINVAL will
>              * be returned if both are specified.
>              * This flag is not support stdio, cwd, env and stdio_count options in
>              * the uv_process_options_t struct, and will get UV_EINVAL if any of them
>              * are specified.

Reviewer: ```suggestion
             * This flag is mutually exclusive with UV_PROCESS_DETACHED. UV_EINVAL is
             * returned when both are specified.
             * This flag is mutually exclusive with the stdio, cwd, env and stdio_count options in
             * the uv_process_options_t struct. UV_EINVAL is returned when any of them
             * are specified.
```
